encass-annotations/src/main/gateway/config/jdbc-connections.yml

```diff
@@ -0,0 +1,18 @@
+jdbc_con1:
```

Comment:
name the connection as **internal-users-connection**

Reply:
Done